File: app/src/main/java/com/akrivonos/a2chparser/pojomodel/boardmodel/BoardTheme.java
package com.akrivonos.a2chparser.pojomodel.boardmodel;

import com.google.gson.annotations.Expose;

import java.util.List;

public class BoardTheme {

    @Expose
    private String boardThemeName = null;
    @Expose
    private List<BoardConcrete> boardConcretes = null;

    public String getBoardThemeName() {
        return boardThemeName;
    }

    public void setBoardThemeName(String boardThemeName) {
        this.boardThemeName = boardThemeName;
    }

    public List<BoardConcrete> getBoardConcretes() {
        return boardConcretes;
    }

    public void setBoardConcretes(List<BoardConcrete> boardConcretes) {
        this.boardConcretes = boardConcretes;
    }


}

File: app/src/main/java/com/akrivonos/a2chparser/pojomodel/boardmodel/BoardTheme.kt
package com.akrivonos.a2chparser.pojomodel.boardmodel

import com.google.gson.annotations.Expose

class BoardTheme(boardConcretes: List<BoardConcrete>?) {

    @Expose
    var boardThemeName: String? = null
    @Expose
    var boardConcretes: List<BoardConcrete>? = null

    init {
        if (boardConcretes != null && boardConcretes.isNotEmpty()) {
            boardThemeName = boardConcretes[0].category
            this.boardConcretes = boardConcretes
        }
    }


}
